UploadDaemon/UploadTask.cs

```diff
@@ -156,4 +161,4 @@ private static async Task ProcessMethodCoverage(TraceFile trace, Archiver archiv
             }
         }
     }
-}
+}
```

Comment:
Missing newline

Reply:
I changed the formatter settings to insert newlines